app/controllers/app_controller.rb

```diff
@@ -0,0 +1,2 @@
+class AppController < ApplicationController
+end
```

Comment:
i think you mean to say ApiController here?

Reply:
We are deciding to leave the app layer for future purposes in case we render the navigator bars in React.